crates/uv-scripts/src/lib.rs

```diff
@@ -426,6 +426,7 @@ pub struct ToolUv {
     #[serde(flatten)]
     pub top_level: ResolverInstallerSchema,
     pub override_dependencies: Option<Vec<uv_pep508::Requirement<VerbatimParsedUrl>>>,
+    pub exclude_dependencies: Option<Vec<uv_normalize::PackageName>>,
```

Comment:
it looks like this is not handled in RequirementsSpecification.from_pep723_metadata 🤔